server/src/test/java/org/opensearch/repositories/RepositoriesServiceTests.java

```diff
@@ -353,6 +354,18 @@ public void cloneShardSnapshot(
 
         }
 
+        @Override
+        public void cloneRemoteStoreIndexShardSnapshot(
+            SnapshotId source,
+            SnapshotId target,
+            RepositoryShardId shardId,
+            String shardGeneration,
+            RemoteStoreLockManagerFactory remoteStoreLockManagerFactory,
+            ActionListener<String> listener
+        ) {
+
```

Comment:
why empty ?

Reply:
Empty implementation for `TestRepository` implementing `Repository` interface.